Brokerages/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="IKVM-WithExes" version="7.3.4830.1" />
-  <package id="Krs.Ats.IBNet" version="9.66.0.31" targetFramework="net45" />
+  <package id="Krs.Ats.IBNet" version="9.66.0.30" targetFramework="net45" />
```

Comment:
Krs.Ats.IBNet should go away and your TWSLib will take its place, please rename the TWS lib to a more appropriate name reflecting the name of the ib github project. (TWS is just Trader Work Station -- we should make sure this works with the Gateway as well)